examples/allocation_benchmark.c

```diff
@@ -5,7 +5,7 @@
 
 int main() {
     volatile uint64_t * tmp;
-    mmtk_gc_init(1024*1024*1024);
+    mmtk_init(24*1024*1024);
```

Comment:
This is 24 (twenty four) megabytes, but the loop below attempted to allocate 800 megabytes of objects.  Is this intentional? Did you mean `1024*1024*1024`?